client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -118,6 +118,7 @@ describe( 'EditGravatar', function() {
 					<EditGravatar
 						translate={ noop }
 						user={ user }
+						receiveImage={ noop }
```

Comment:
These functions don't need a default value since they're added by `mapDispatchToProps`, and they only record analytics events. I hesitate to add default values only to make the tests neater, since in general we avoid changing app code for tests' sake in Calypso.